10.md

```diff
@@ -1,15 +1,22 @@
 NIP-10
 ======
 
-
-On "e" and "p" tags in Text Events (kind 1).
---------------------------------------------
+Text Notes and Threads
+----------------------
 
 `draft` `optional`
 
+This NIP defines `kind:1` as a simple plaintext note.
+
 ## Abstract
 This NIP describes how to use "e" and "p" tags in text events, especially those that are replies to other text events.  It helps clients thread the replies into a tree rooted at the original event.
 
+The `.content` property contains a text. 
+
+`e` and `p` tags can be used to define note threads, replies and mentions. 
+
+Markup languages such as markdown and HTML SHOULD NOT be used. 
```

Comment:
There are lots of issues in kind 1 notes. For instance, there is a new client that is pasting base64 images right into the text to avoid using an image server. The `.content` looks like this: 

```
Lorem ipsum....

data:image/svg+xml;base64,PD94bWwg..[10000 chars later]..go8L3N2Zz4K

Lorem ipsum....
```

In that case, clients are supposed to parse the `data:image/svg+xml;base64,` segment into an image.

Reply:
Maybe the Kind 1 spec should be updated to allow only human-readable text and embed codes for other Nostr events.  Anything else seems rather egregious.  Updating the spec in that regard might be the work of another PR, though.

Ideally clients would simply reject notes like the example you gave, but that requires the clients to parse the `.content` of each note, which is likely impracticable for some.

Reply:
Yeah, we can keep adding more guidance in the NIP so people know what's expected and what isn't. There is a lot to add. The more specific, the better it gets for interoperability. 

Then people who want new formats can easily just add a new kind for those formats.

Reply:
> In that case, clients are supposed to parse the `data:image/svg+xml;base64,` segment into an image.

Was that the problem with that weird note that crashed your client? The image section looked really odd...

Reply:
Well, this one doesn't crash but it was a VERY long note.